pubspec.yaml

```diff
@@ -9,6 +9,8 @@ environment:
 dependencies:
   analyzer: ">=0.30.0 <0.31.0"
   barback: "^0.15.0"
+  built_redux: ^6.1.0
+  built_value: ^4.2.0
```

Comment:
Is this dependency needed?

Reply:
Hmm I guess it's only needed for tests, will move